### PR TITLE
Universal instance type

### PR DESCRIPTION
**Pull request by nmenardg**
https://www.pivotaltracker.com/story/show/126630419
https://www.pivotaltracker.com/story/show/128009829

### CODE REVIEW

File: nefertari_sqla/documents.py [hunk 1]
```diff
@@ -681,41 +682,54 @@ def _update_many(cls, items, params, request=None, synchronize_session='fetch',
...
+        #         updated_items = upd_queryset.all() if return_documents is True else []
+        #     else:
+        #         items_count = len(items)
+        #
+        #         for item in items:
+        #             item.update(params, request)
+        #
+        #         updated_items = items if return_documents is True else []
+        #
+        # return {"count": items_count, "items": updated_items}
```
Comment (nmenardg) [inline nefertari_sqla/documents.py]: Take this commented to another pull request, it's optimized but not tested.

**Comment by oleduc** [comment]
Approved.

**Comment by nikitagromov** [comment]
Add few unit-tests for covering new behaviour, please

**Comment by oleduc** [comment]
@nikitagromov Not quite sure how relevant that is. This falls in the category high maintenance test that don't provide much.

http://blog.stevensanderson.com/2009/08/24/writing-great-unit-tests-best-and-worst-practises/

**Comment by nikitagromov** [comment]
@oleduc In this changes, I mean cover function get_view, that we will be sure that this function returns us dictionary with DocumentType, and check test "def test_to_dict".

**Comment by nmenardg** [comment]
I have to agree with Olivier here. That'd be a change detector. It doesn't validate any algorithm.

**Pull request merged**